CMakeLists.txt

```diff
@@ -118,7 +118,7 @@ include(CMakePackageConfigHelpers)
 
 write_basic_package_version_file(
   "${PROJECT_BINARY_DIR}/campConfigVersion.cmake"
-  VERSION 0.2.3
+  VERSION ${PACKAGE_VERSION}
```

Comment:
I think this should be `$PROJECT_VERSION` (although if the VERSION argument is not used, then PROJECT_VERSION is used by default afaik)

Reply:
Thanks for that, you're right.

Reply:
```suggestion
  VERSION ${PROJECT_VERSION}
```